src/Player/CapFunction.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+class HackCap;
+class PlayerAnimator;
+
+namespace CapFunction {
```

Comment:
Do we know if this is a namespace, or a class with static functions?

Reply:
nope. dont know if its possible to know either, same case as GameDataFunction